source/imagesegmentation.rst

```diff
@@ -62,17 +72,17 @@ When we execute this workflow we are presented with :numref:`dtp_cp_seg_tsk2_ini
    
    Heart segmentation interface initial state
    
-In :numref:`dtp_cp_seg_tsk2_init` we can see on the left a toolbox that allows us to change the state of this segmentation tool, on the right hand side we can see a three-dimensional view of the two sets of DICOM images.  To create this view we have used the
+In :numref:`dtp_cp_seg_tsk2_init` we can see on the left a toolbox that allows us to change the state of this segmentation tool, on the right hand side we can see a three-dimensional view of the two sets of DICOM images.  To create this view we have used the the following information from the DICOM header:
 
  * Pixel spacing
  * Image orientation patient
  * Image position patient
  * Rows
  * Columns
  
-information from the DICOM header.  This has placed each image plane in the machine or magnet coordinate system.  In the images we are using you will see lines across the image picture, this comes from the saturated MR signals so that we can track myocardial motion. In the images that we see we have straight saturated bands indicating that these are the reference images.
+This has placed each image plane in the machine or magnet coordinate system.  In the images we are using, you will see lines across the image picture, this comes from the saturated MR signals so that we can track myocardial motion. In the images that we see we have straight saturated bands indicating that these are the reference images.
 
-From the view toolbox on the left-hand side we can show the image planes and from the file for box we can load and save our progress. The done button is also in the file toolbox for when we are finished segmenting.
+From the *view* toolbox on the left-hand side we can show the image planes and from the *file* toolbox we can load and save our progress. The done button is also in the file toolbox for when we are finished segmenting.
```

Comment:
The *view* toolbox could probably be better labelled with a :guilable: Sphinx markup rather than italics (and others).